src/commands/BugCommand.ts

```diff
@@ -15,6 +16,10 @@ export default class BugCommand extends SlashCommand {
 		);
 
 	public async run( interaction: ChatInputCommandInteraction ): Promise<boolean> {
+		if ( interaction.channel === null ) return false;
+
+		if ( ChannelConfigUtil.mentionsDisabled( interaction.channel ) ) return false;
```

Comment:
This doesn't seem right, wouldn't this rather be covered by the ~no-command flag?

Reply:
👍